internal/initload/initload.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// Package initload is an internal package with helper for data and policy loading during initialization.
+package initload
```

Comment:
Since `initload` is so generic would it make sense to nest it under like `internal/runtime/init` or something? I guess for someone looking through the OPA source I could see this being kind of confusing without more context. In the context of this PR it makes sense that it is plugin manager and runtime initialization helpers to load policies and files.. But there are a _lot_ of things that it could be providing helpers to initialize, we probably wouldn't want to push in stuff for like initializing a topdown query or prometheus registry or some other random thing in here too, hence the slightly more specific naming/directory path to scope it a little bit more.

Reply:
Makes sense.